pkg/model/iam/iam_builder.go

```diff
@@ -101,21 +101,23 @@ func (b *IAMPolicyBuilder) BuildAWSIAMPolicy() (*IAMPolicy, error) {
 			Resource: wildcard,
 		})
 
-		// No longer needed in 1.3
-		//p.Statement = append(p.Statement, &IAMStatement{
-		//	Effect: IAMStatementEffectAllow,
-		//	Action: []string{ "ec2:AttachVolume" },
-		//	Resource: []string{"*"},
-		//})
-		//p.Statement = append(p.Statement, &IAMStatement{
-		//	Effect: IAMStatementEffectAllow,
-		//	Action: []string{ "ec2:DetachVolume" },
-		//	Resource: []string{"*"},
-		//})
+		p.Statement = append(p.Statement, &IAMStatement{
```

Comment:
Can we override these changes, as I would rather have this as a non-breaking change.  Also, have we tested different route53 controllers against this PR.  Not sure what impact this will have, on existing deployments.